VideoLocker/src/main/java/org/edx/mobile/view/adapters/DownloadEntryAdapter.java

```diff
@@ -96,16 +132,29 @@ public void onItemClick(AdapterView<?> arg0, View arg1, int position, long arg3)
     public abstract void onDeleteClicked(DownloadEntryAdapter.Item model);
 
     public interface Item {
-        long getSize();
-
+        @NonNull
         String getTitle();
 
+        @NonNull
         String getDuration();
 
-        String getDownloaded();
+        @NonNull
+        Status getStatus();
+
+        /**
+         * @return Total download size in bytes, or null if size is not yet known
+         */
+        @Nullable
+        Long getTotalByteCount();
 
-        int getStatus();
+        long getDownloadedByteCount();
 
         int getPercent();
```

Comment:
I guess this could also be renamed to `getPercentDownloaded()` along with the other similar naming changes.